content/Assets/Styles/_fonts.scss

```diff
@@ -0,0 +1,38 @@
+/* manuale-regular - latin */
+@font-face {
+    font-family: 'Manuale';
+    font-style: normal;
+    font-weight: 400;
+    src: local(''),
+        url('../Fonts/manuale-v8-latin-regular.woff2') format('woff2'),
+        /* Chrome 26+, Opera 23+, Firefox 39+ */
+            url('../Fonts/manuale-v8-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* manuale-700 - latin */
+@font-face {
+    font-family: 'Manuale';
+    font-style: normal;
+    font-weight: 700;
+    src: local(''), url('../Fonts/manuale-v8-latin-700.woff2') format('woff2'),
+        /* Chrome 26+, Opera 23+, Firefox 39+ */
+            url('../Fonts/manuale-v8-latin-700.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
```

Comment:
All looks good to me, but one thought here: Wouldn't we want to include an italic version for both fonts? Given that CMS users have the option, if we don't provide one the browser will attempt a faux-italic which usually looks gross.

Reply:
Good shout.

Reply:
This conversation was resolved but I can't see any change, is that our intent? If it's deliberate that's cool.

Reply:
I've moved it back to unresolved, must have marked it as resolved by mistake.